app/helpers/pagination_helper.rb

```diff
@@ -55,6 +53,8 @@ def need_letter_pagination_links?(pages)
       (!pages.used_letters || pages.used_letters.length > 1)
   end
 
+  LETTERS = ("A".."Z").freeze
```

Comment:
@mo-nathan / @pellaea : What's the best/most conventional place for constants in these kinds of files:
- top of module
- top of the section (i.e., here just above `def pagination_letters`)
- order of use (which is where I ended up sticking it)

Reply:
If I understand your terms, I'm most used to "top of module".  "Top of the section" is also reasonable.  "Order of use" (meaning after they get referenced) is surprising to me.